common/fe-ste-models.go

```diff
@@ -119,6 +119,19 @@ func (DeleteSnapshotsOption) None() DeleteSnapshotsOption    { return DeleteSnap
 func (DeleteSnapshotsOption) Include() DeleteSnapshotsOption { return DeleteSnapshotsOption(1) }
 func (DeleteSnapshotsOption) Only() DeleteSnapshotsOption    { return DeleteSnapshotsOption(2) }
 
+type SetPropertiesAPIOption uint32 // [0000000000...32 times]
+
+var ESetPropertiesAPIOption = SetPropertiesAPIOption(0)
+
+// functions to set values
+func (SetPropertiesAPIOption) None()        { ESetPropertiesAPIOption |= 0 }
```

Comment:
Generally speaking, global variables are considered bad practice. Instead, return bitflags to be ORed into place, as this looks like it's supposed to be a bitflag enum.

E.g. I should be writing `ESetPropertiesAPIOption.SetTier | ESetPropertiesAPIOption.SetMetadata`

Reply:
Upon second viewing, throughout your code, you choose to use SetPropertiesAPIOption two ways:

1) As a local variable in the STE
2) As a global variable in the frontend

Reply:
resolved